.github/workflows/build-electron.yml

```diff
@@ -88,9 +91,9 @@ jobs:
       env:
         CLOUDSDK_PYTHON: ${{env.pythonLocation}}\python.exe
       run: |
-        gsutil -m rsync dist/ "${{ env.BUCKET }}/${{ env.PKG_VERSION }}_${{ env.GIT_REV }}/"
+        gsutil -m rsync dist/ "${{ env.BUCKET }}/${{ env.VERSION }}/"
 
     - name: Deploy artifacts to GCS - macOS
       if: github.event_name != 'pull_request' && matrix.os == 'macos-latest'
       run: |
-        gsutil -m rsync dist/ "${{ env.BUCKET }}/${{ env.PKG_VERSION }}_${{ env.GIT_REV }}/"
+        gsutil -m rsync dist/ "${{ env.BUCKET }}/${{ env.VERSION }}/"
```

Comment:
It looks like we have two places where `git describe --tags` is used to get the version, one here directly in the actions workflow YML, and once again in `build.js` when it's written to `electron-builder.env`.  Would it make sense to consolidate the source of truth for the version string?  Like, maybe what's here as `env.VERSION` can be drawn from `electron-builder.env` somehow?  It'd be frustrating down the road to forget to change the version string in one place but not the other and then we don't know where the binaries ended up on GCP.

Reply:
Yes, good idea. I grepped for it. And I also renamed some variables to always call this string a "version" instead of a git rev since that's a little less accurate for what it is.